### PR TITLE
REALMC-8432: Set correct file permissions for static files

### PR DESCRIPTION
**Pull request by UnicodeSnowman**
* 'rw' should be sufficient here, don't need executable

### CODE REVIEW

File: internal/local/hosting.go [hunk 1]
```diff
@@ -23,12 +23,12 @@ const (
...
-		api.HeaderContentLanguage:         struct{}{},
-		api.HeaderContentEncoding:         struct{}{},
-		api.HeaderCacheControl:            struct{}{},
-		api.HeaderWebsiteRedirectLocation: struct{}{},
+		api.HeaderContentType:             {},
+		api.HeaderContentDisposition:      {},
+		api.HeaderContentLanguage:         {},
+		api.HeaderContentEncoding:         {},
+		api.HeaderCacheControl:            {},
+		api.HeaderWebsiteRedirectLocation: {},
```
Comment (UnicodeSnowman) [inline internal/local/hosting.go:31]: my editor did this 🤷

**Comment by UnicodeSnowman** [comment]
FYI gonna fix this in `1.3.x` too

File: internal/local/hosting.go (hunk 1 above)
Comment (makesitgo) [reply internal/local/hosting.go:31]: i learned something today... neat!

**Review by makesitgo — APPROVED** [review]
🆒  beans

**Pull request merged**